spec/controllers/idv/hybrid_handoff_controller_spec.rb

```diff
@@ -157,6 +157,7 @@
       it 'sends analytics_submitted event for hybrid' do
         put :update, params: { doc_auth: { phone: '202-555-5555' } }
 
+        expect(subject.idv_session.phone_for_mobile_flow).to eq('+1 202-555-5555')
```

Comment:
Is there a good feature spec to test this?

Reply:
Are we checking that the phone number is shown correctly on the desktop screen in the hybrid handoff? 

Maybe around here?

https://github.com/18F/identity-idp/blob/66d58b5f40f550f223da4d5989675e70c9c83d4e/spec/features/idv/hybrid_mobile/hybrid_mobile_spec.rb#L30

Reply:
that was perfect, thank you!